### DISCUSSION TITLE
Nodejs as dev dependency

### DISCUSSION THREAD

**Discussion by luisfilsilva** [artilleryio/artillery#1225] [Artillery]
Hi experts,

This is a newbie question.. I want to run artillery as dev dependency on nodejs:

npm install -D artillery@latest

However, I'm not really sure how I can use it after the installation. Can I call artillery from my own nodejs project (instead of command line)?

Thanks!!
Luís

**Comment by hassy** [comment]
hi @luisfilsilva - when installed as a dependency, the `artillery` CLI can be run from `node_modules/.bin/artillery` in the project directory. You can also run it with `$(npm bin)/artillery` from the directory (on Linux/macOS)

**Reply by luisfilsilva** [reply]
Thanks Hassy!! Works!
